### ISSUE TITLE
A canceled request to changes spams the log output

### ISSUE THREAD

**Issue by vincnetas** [apache/couchdb#2041]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

Canceling _changes request produces [error] level log in CDB.

With big amount of users on unreliable connections listening for changes in CDB, log gets flooded with error messages which are actually not errors. This affects also _stats endpoint which reports number of 500 errors, which makes monitoring for real errors impossible.

## Steps to Reproduce

Start docker :

```
docker run \
-d --name couch_db \
-p 5984:5984 \
--env COUCHDB_USER=admin \
--env COUCHDB_PASSWORD=thisiscouchdb \
--env NODENAME=127.0.0.1 couchdb:2.3.1
```

Create users DB 
`curl -X PUT http://127.0.0.1:5984/_users -u admin`

Initiate request for changes
`curl "http://127.0.0.1:5984/_users/_changes?feed=longpoll&heartbeat=10000&since=now&limit=25" -u admin`

Cancel it with Ctrl+C

After 30 seconds error message gets logged (docker logs couch_db)

```
[error] 2019-05-31T10:29:16.271783Z couchdb@127.0.0.1 <0.3233.0> 717801f155 req_err(1125717067) unknown_error : normal
    [<<"chttpd:catch_error/3 L356">>,<<"chttpd:handle_req_after_auth/2 L322">>,<<"chttpd:process_request/1 L303">>,<<"chttpd:handle_request_int/1 L243">>,<<"mochiweb_http:headers/6 L128">>,<<"proc_lib:init_p_do_apply/3 L247">>]
```

## Expected Behaviour

Canceling _changes request should not produce error level log message

## Your Environment

2.3.1 official couchdb docker image

* CouchDB Version used: 2.3.1
* Browser name and version: curl
* Operating System and version: osx

**Comment by vincnetas** [comment]
Identical issue was closed as wont implement here (for older version) : https://issues.apache.org/jira/browse/COUCHDB-3161

**Comment by nickva** [comment]
Thanks for reaching out @vincnetas. 

This is a duplicate of https://github.com/apache/couchdb/issues/1600 and should already be fixed in #3102
